src/constants/constants.json

```diff
@@ -5,7 +5,16 @@
 	
 	"Vision":{
 		"visionAddress": "224.5.23.2",
-		"visionPort": 10002
+		"visionPort": 10002,
+		"lossFilterTime": 300,
+		"noiseFilterTime": 300,
+		"useGeometryFromCamera": false
```

Comment:
Set these new parameters to change filter times and to use geometry from camera if needed. For default, it is loading the geometry data in the field pointer passed in Suassuna class.